src/stories/containers/EndgamePhaseOneProgress/components/RoadmapTimeline/RoadmapTimeline.tsx

```diff
@@ -0,0 +1,206 @@
+import styled from '@emotion/styled';
+import { useMediaQuery } from '@mui/material';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import MilestoneCard from '../MilestoneCard/MilestoneCard';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+const RoadmapTimeline = () => {
+  const { isLight } = useThemeContext();
+  const isMobile = useMediaQuery(lightTheme.breakpoints.down('tablet_768'));
+  const viewAll = false;
+  const milestones = Array.from({ length: 8 });
+
+  const up = milestones.length <= 4 ? milestones : milestones.filter((_, i) => i % 2 === 0);
+  const down = milestones.filter((_, i) => i % 2 !== 0);
+
+  const shouldAddPadding = milestones.length % 2 === 0 && milestones.length > 4;
+
+  return (
+    <div>
+      {isMobile ? (
+        <MobileTimeline isLight={isLight}>
+          <MilestoneCard />
+          <MilestoneCard />
+          <MilestoneCard />
+          <MilestoneCard />
+          <MilestoneCard />
+
+          <ButtonBox>
+            <Button isLight={isLight}>
+              <span>View all Roadmap</span>
+              <svg
```

Comment:
Its possible to made this a component, also to avoid inline style or you never will use this component again

Reply:
This is not going to be reused and it is a really simple svg so there's no need to create a new component. The style in this case is to rotate the SVG and it depends on a state vars, this way is more simple and readable

Reply:
LGTM